src/providers/martian/index.ts

```diff
@@ -0,0 +1,18 @@
+import { ProviderConfigs } from '../types';
+import MartianAPIConfig from './api';
+import {
+  MartianChatCompleteConfig,
+  MartianChatCompleteResponseTransform,
+} from './chatComplete';
+
+const MartianConfig: ProviderConfigs = {
+  api: MartianAPIConfig,
+  chatComplete: MartianChatCompleteConfig,
+  responseTransforms: {
```

Comment:
For OpenAI compliant providers, you can reuse the openai-base provider's function to avoid writing redundant code.

Please check inference-net provider integration as a reference on how to use the base provider:
https://github.com/Portkey-AI/gateway/blob/213273d9c0451203c2c18bc38279b48259bb4ee5/src/providers/inference-net/index.ts#L9-L11